src/main/java/com/chat/yourway/config/RedisConfig.java

```diff
@@ -13,6 +13,9 @@
 @Configuration
 public class RedisConfig {
 
+  @Value("${spring.profiles.active}")
+  private String profile;
+
   @Value("${spring.data.redis.host}")
```

Comment:
if u use more than one @Value in one class or if u use more than once the same @Value property -> move to setting class (@ConfigurationProperties)
https://www.baeldung.com/configuration-properties-in-spring-boot